.github/workflows/build-and-deploy.yml

```diff
@@ -417,17 +417,43 @@ jobs:
           deployment_id: ${{ steps.deployment.outputs.deployment_id }}
           env_url: ${{ steps.deploy_review.outputs.deploy-url }}
 
-  deploy-main:
-    name: Sequential Deployment
+  deploy-before-prod:
```

Comment:
Cosmetic change: can we use "production" instead of "prod"?

We want to rename everything to production as some point, so this will be less work

Reply:
changed prod to production